users/tokens.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from django.utils import six
+
+
+class TokenGenerator(PasswordResetTokenGenerator):
+    def _make_hash_value(self, user, timestamp):
+        return six.text_type(user.pk) + six.text_type(timestamp) + six.text_type(user.is_verified)
+
+
+verification_token = TokenGenerator()
```

Comment:
nit: This isn't a token, it's a token generator. I prefer `verification_token_generator`